apis/apis.json

```diff
@@ -477,7 +477,7 @@
       "Google.Cloud.Spanner.Admin.Database.V1": "project",
       "Google.Cloud.Spanner.Admin.Instance.V1": "project",
       "Microsoft.EntityFrameworkCore.Relational": "2.0.1",
-      "Grpc.Core": "default"
+      "Grpc.Core": "1.7.1"
```

Comment:
Is there a reason this is now fixed?

Reply:
Because its fixed for the ADO.NET layer.  That change was done a while ago and I cannot recall the reason, but it fixed an issue with going GA in spanner.  Something like the GA bits had to depend only on GA dependencies IIRC